packages/core/parcel-bundler/test/server.js

```diff
@@ -1,9 +1,11 @@
 const assert = require('assert');
 const path = require('path');
 const fs = require('@parcel/fs');
+const logger = require('@parcel/Logger');
```

Comment:
any guidance on how to import that properly? Seems to work locally, but not in CI.

Reply:
You have a typo it's lowercase `l`, `'@parcel/logger'`